editor.planx.uk/src/setupTests.ts

```diff
@@ -1 +1,3 @@
 import "@testing-library/jest-dom";
+// import "@testing-library/jest-dom/vitest"; << https://stackoverflow.com/questions/77611978/invalid-chai-property-in-vitest
+import "vitest-axe/extend-expect";
```

Comment:
Bug here! Axe tests currently throwing `Chai property not defined` error in thread, but haven't gotten this solution working yet either